src/ng4fittext.directive.ts

```diff
@@ -26,6 +26,14 @@ export class Ng4FittextDirective implements AfterViewInit {
         }
     };
 
+    ngOnChanges(changes: SimpleChanges) {
```

Comment:
make this a public method

Reply:
Sorry for my question, but why that?

https://angular.io/api/core/OnChanges